akka-remote/src/main/scala/akka/remote/artery/Association.scala

```diff
@@ -476,19 +477,20 @@ private[remote] class Association(
                 clearOutboundCompression()
                 clearInboundCompression(u)
                 // end delivery of system messages to that incarnation after this point
-                send(ClearSystemMessageDelivery, OptionVal.None, OptionVal.None)
+                send(ClearSystemMessageDelivery(current.incarnation), OptionVal.None, OptionVal.None)
                 // try to tell the other system that we have quarantined it
                 sendControl(Quarantined(localAddress, peer))
                 startQuarantinedIdleTimer()
               } else
                 quarantine(reason, uid) // recursive
             }
           case Some(peer) ⇒
-            log.debug(
+            log.info(
```

Comment:
Was it intentional to make these info? I think given how infrequent they are it is a good idea

Reply:
yes, I think it could be useful to see this case